internal/plugin/connectors/http/generic/README.md

```diff
@@ -103,6 +108,41 @@ powerful transformation features.  You can use `printf` for formatting and
 compose functions using pipes `|`.  See the text template package docs linked
 above for detailed information on these and other features.
 
+### `queryParams`
+
+Like `headers`, this is another key section. The `queryParams` section
+is used to generate a query string, which is appended to your existing URL
+without replacing any existing query parameters.
+
+The _keys_ of the queryParams are defined by the yaml keys.  In the examples
+above, these query parameter key is `location`.
+
+The query parameter _values_ are defined using a [Go text
+template](https://golang.org/pkg/text/template/), as defined in the
+`text/template` package.
+
+In the above example, let us say that your request URL looks like the following,
+
+```
+http://anything.com/foo?fruit=apple
+```
+
+After proxying through secretless, your request URL would look like the following,
+
+```
+http://anything.com/foo?location=valueofaddress&fruit=apple
+```
+
+You can refer to your credentials in this template using the credential name
+preceded by a `.` (eg, `.address` will refer to the credential
+`address`). At runtime, Secretless will replace these
+credential references with your real credentials.
+
+As you can see in the Basic auth example, the `text/template` package has
+powerful transformation features.  You can use `printf` for formatting and
+compose functions using pipes `|`.  See the text template package docs linked
```

Comment:
```suggestion
you can compose functions using pipes `|`.  See the text template package docs linked
```
not having the "you can" at the start here led me to try to understand "formatting and compose functions" as a concept